apps/landing/features/home/HeadingSection/SignUpModal/InputDropdown/index.tsx

```diff
@@ -60,10 +60,16 @@ const InputDropdown: FC<Props> = ({
 		}
 	}, [isDropped]);
 
+	const errorStyle = {
+		borderWidth: 1,
+		borderRadius: 15,
+		borderColor: '#E34237',
+	};
+
 	return (
 		<View style={styles.container}>
 			<Text style={styles.title}>{title}</Text>
-			<View style={styles.contentContainer}>
+			<View style={[styles.contentContainer, error ? errorStyle : {}]}>
```

Comment:
when having error, it shows error style with borderWidth change, I was wondering if It might effect the layout

Reply:
> 

i don't get what you mean

Reply:
I mean, when not having error, the `View` will be showed without border. and with error, It shows with border. This make the layout jerky